src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
+    static String sectionDivider = "____________________________________________________________";
+    public static void greetUser(){
+        String greeting = "\t" + sectionDivider + "\n"
+                + "\tHello! I'm Duke. \n"
+                + "\tWhat can I do for you? \n"
+                + "\t" + sectionDivider;
+        System.out.println(greeting);
+    }
+    public static void signOff(){
+        String sign_off = "\t" + sectionDivider + "\n"
+                + "\tBye. Hope to see you again soon!\n"
+                + "\t" + sectionDivider;
+        System.out.println(sign_off);
+    }
+
+    static int numberOfTasks = 0;
+    static Task tasks[] = new Task[100];
+
+    public static void addTask(String taskDescription){
+        System.out.println("\tGot it. I've added this task: ");
+        Task t;
+        if (taskDescription.contains("todo")) {
+            t = new Todo(taskDescription.substring(5));
```

Comment:
Can consider declaring the constant '5'. I've found the same issue in some other places as well.